db/schema.rb

```diff
@@ -755,6 +756,7 @@
     t.index ["organization_id"], name: "index_invoices_on_organization_id"
     t.index ["payment_overdue"], name: "index_invoices_on_payment_overdue"
     t.index ["sequential_id"], name: "index_invoices_on_sequential_id"
+    t.index ["status"], name: "index_invoices_on_status"
```

Comment:
why this?

Reply:
I added it last week and I think it was missing in the schema!